whylog/tests/tests_config/test_investigation_plan_utils.py

```diff
@@ -0,0 +1,109 @@
+import itertools
+import os.path
+from unittest import TestCase
+
+from whylog.config import YamlConfig
+from whylog.front import FrontInput
+
+path_test_files = ['whylog', 'tests', 'tests_config', 'test_files', 'test_investigation_plan_files']
+
+
+class TestBasic(TestCase):
```

Comment:
You've named file "...utils.py", but have contained tests inside. Please change file name or move tests from this file to another one.